incidents/2023-02-13--epp--prod-server-git-sync-fails.md

```diff
@@ -0,0 +1,40 @@
+# 2023-02-13 EPP Prod outage
+
+## Description
+
+EPP preprint pages were throwing a 500 error. The cause of the 500 was due to the EPP server pod being not ready, and not routable from EPP client
+
+## Timeline
+
+2023-02-13:
+
+* 0717: Routine cluster component upgrade merged in the `epp-flux-cluster`
+* 0734: prometheus monitoring sent a KubPodCrashLooping #cluster-alerts
```

Comment:
I've now joined the slack channel. Lot's of noise on there but epp currently appears to be up.